BioArchLinux/snpeff/lilac.yaml

```diff
@@ -0,0 +1,11 @@
+maintainers:
+  - github: kbipinkumar
+    email: bipin@ccmb.res.in
+build_prefix: extra-x86_64
+post_build_script:
+  git_add_files('PKGBUILD')
+  git_commit()
+  update_aur_repo()

```

Comment:
so you should remove this line, as it causes cyclic updates.
When you push an update to aur, it fetches update from aur (`update_on: aur`) and then push updates to aur `update_aur_repo()`, and fetch updates from aur ....

Reply:
i have updated `lilac.yaml` with suggested github repo as update source. 

also pls take a look at the perl dependencies in the request and review them whenever you have time